src/main/java/roomescape/reservation/ReservationService.java

```diff
@@ -28,12 +62,50 @@ private ReservationRequest replaceNameIfEmpty(ReservationRequest request, LoginM
     }
 
     public void deleteById(Long id) {
-        reservationDao.deleteById(id);
+        reservationRepository.deleteById(id);
     }
 
     public List<ReservationResponse> findAll() {
-        return reservationDao.findAll().stream()
-                .map(it -> new ReservationResponse(it.getId(), it.getName(), it.getTheme().getName(), it.getDate(), it.getTime().getValue()))
+        return reservationRepository.findAll().stream()
+                .map(reservation -> new ReservationResponse(
+                        reservation.getId(),
+                        reservation.getName(),
+                        reservation.getTheme().getName(),
+                        reservation.getDate(),
+                        reservation.getTime().getTime()
+                ))
                 .toList();
     }
+
+    public List<MyReservationResponse> findMyReservation(LoginMember loginMembmer) {
+        List<Reservation> reservations = reservationRepository.findByMemberId(loginMembmer.getId());
+        List<WaitingWithRank> waitings = waitingRepository.findWaitingsWithRankByMemberId(loginMembmer.getId());
+
+        List<MyReservationResponse> myReservationResponses = new ArrayList<>();
+        reservations.forEach(
+                reservation -> myReservationResponses.add(
+                        new MyReservationResponse(
+                                reservation.getId(),
+                                reservation.getTheme().getName(),
+                                reservation.getDate(),
+                                reservation.getTime().getTime(),
+                                "예약"
+                        )
+                )
+        );
+
+        waitings.forEach(
+                waitingWithRank -> {
+                    Waiting waiting = waitingWithRank.getWaiting();
+                    myReservationResponses.add(
+                            new MyReservationResponse(
+                                    waiting.getId(),
+                                    waiting.getTheme().getName(),
+                                    waiting.getDate(),
+                                    waiting.getTime().getTime(),
+                                    waitingWithRank.getRank() + "번째 예약대기"));
+                });
```

Comment:
```suggestion
List<MyReservationResponse> myReservationResponses = waitings.stream()
    .map(waitingWithRank -> {
        Waiting waiting = waitingWithRank.getWaiting();
        return new MyReservationResponse(
            waiting.getId(),
            waiting.getTheme().getName(),
            waiting.getDate(),
            waiting.getTime().getTime(),
            waitingWithRank.getRank() + "번째 예약대기"
        );
    })
    .collect(Collectors.toList());
```

여기도 stream 사용이 가능합니다